src/CommunityToolkit.Maui.Core/Views/Expander/PlatformView/MauiExpanderExtensions.shared.cs

```diff
@@ -0,0 +1,73 @@
+﻿using CommunityToolkit.Maui.Core.Views;
+using Microsoft.Maui.Platform;
+#if WINDOWS
+using MauiExpander = Microsoft.UI.Xaml.Controls.Expander;
+#endif
+namespace CommunityToolkit.Maui.Core.Extensions;
+
+/// <summary>
+/// Extension methods to support <see cref="IExpander"/>
+/// </summary>
+public static partial class MauiExpanderExtensions
+{
+#if ANDROID || IOS || MACCATALYST || WINDOWS
+
+
+	/// <summary>
+	/// Set Header
+	/// </summary>
+	/// <param name="mauiExpander"><see cref="MauiExpander"/></param>
+	/// <param name="header">header</param>
+	public static void SetHeader(this MauiExpander mauiExpander, IView header, IMauiContext? context)
```

Comment:
Can we check if the MauiContext is null on the caller side? That way the we can remove the `?` from `IMauiContext`